### ISSUE TITLE
[Feature]: Changing key sorting order in UI tables

### ISSUE THREAD

**Issue by mattdl-meta** [wandb/wandb#4281]
### Description

When doing a manual gridsearch, it would be really useful to enable changing the sorting order easily.
Sorting now requires adding various fields, but the order of keys on which the table is sorted is not easily configurable.

For example, I want to sort on learning rate, then batch size, then weight decay strength.
Right now I have to remove and re-add the different sorting keys until I have the desired key-order. 
This is often on a trial-and-error basis and requires removing all keys and re-adding them in the right order.

### Suggested Solution

Having a click and drag, or arrows to change the order of the sorting entries in the UI would be much more convenient.

**Comment by nate-wandb** [comment]
Hi @mattdl-meta, thank you for the feature request. This has previously been requested (outside of Github) and has been submitted to our engineering team. I'll add a +1 and increase the priority on the ticket. 

I can follow up with you here once the engineering team has had a chance to look into this. 

Thank you!
Nate
